backend/src/services/episode/episode.service.ts

```diff
@@ -16,6 +19,14 @@ class Episode {
     return this.#episodeRepository.getAll();
   }
 
+  public async getById(id: string): Promise<TEpisode> {
+    const episode = await this.#episodeRepository.getById(id);
+    if (!episode) {
+      throw new HttpError({ status: HttpCode.NOT_FOUND, message: ErrorMessage.NOT_FOUND });
```

Comment:
Previously you have the more readable error message. Let's replace it with `Entity not found.`